framework/src/outputs/CSV.C

```diff
@@ -142,14 +182,30 @@ CSV::output(const ExecFlagType & type)
 
       it.second.printCSV(getVectorPostprocessorFileName(vpp_name, include_time_suffix), 1, _align);
 
+      if (_create_latest_symlink)
+      {
+        std::string out_latest = _file_base + "_" + short_name + "_LATEST.csv";
```

Comment:
I'm seeing this same pattern of building up the name in a few places in this PR, can you create a method instead to remove code duplication?